Packs/Base/Scripts/script-CommonServer.yml

```
@@ -2097,7 +2097,77 @@ script: >-
 
     /**************************** REGEX FORMATTING end *******************************/
 
+  function getDemistoVersion() {
+      try {
+          return demistoVersion();
+      } catch (ex) {
+          throw 'Failed retriving server version - ' + ex;
+      }
+  }
+
+  function isDemistoVersionGE(version, buildNumber) {
+      var serverVersion = {};
+      try {
+          serverVersion = getDemistoVersion();
+          if (serverVersion.version > version) {
+              return true;
+          } else if (serverVersion.version === version) {
+              if (buildNumber !== undefined) {
```

I would rather we just check if it not falsy. It usually covers edge cases like someone putting there `null`. 
```suggestion
              if (buildNumber) {
```